### PR TITLE
Add translations.properties

### PR DESCRIPTION
**Pull request by ahmadmido**
Add translations.properties file.

### CODE REVIEW

File: translations.properties [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#
+# Copyright (C) 2003-2018 eXo Platform SAS.
```
Comment (thomasdelhomenie) [inline translations.properties]:
```suggestion
# Copyright (C) 2003-2019 eXo Platform SAS.
```

File: translations.properties [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+# License along with this software; if not, write to the Free
+# Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+# 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+#
+
+# Mandatory property that indicate the base directory where each file is located in Crowdin (must end with /).
+baseDir=add-ons/onlyoffice/
+
+# Files
+Onlyoffice.properties=services/src/main/resources/locale/onlyoffice/Onlyoffice_en.xml
```
Comment (thomasdelhomenie) [inline translations.properties]: Did you check if there are useless labels in the files ?

Comment (ahmadmido) [reply translations.properties]: No, i will check now

Comment (ahmadmido) [reply translations.properties]: It's ok for Onlyoffice_en.xml file

**Pull request merged**